pom.xml

```diff
@@ -83,7 +83,7 @@
             <dependency>
                 <groupId>org.jenkins-ci.plugins.workflow</groupId>
                 <artifactId>workflow-step-api</artifactId>
-                <version>999999-SNAPSHOT</version>
+                <version>694.v308f6b_0e2946</version>
```

Comment:
PR should be in draft until this is a release. In the meantime, keep a `TODO` comment linking to the upstream.